ospf6d/ospf6_message.c

```diff
@@ -239,23 +239,24 @@ static void ospf6_hello_recv(struct in6_addr *src, struct in6_addr *dst,
 
 	/* HelloInterval check */
 	if (ntohs(hello->hello_interval) != oi->hello_interval) {
-		if (IS_OSPF6_DEBUG_MESSAGE(oh->type, RECV))
-			zlog_debug("HelloInterval mismatch");
+		zlog_warn("I/F %s HelloInterval mismatch: (my %d, rcvd %d)",
```

Comment:
I think it would be useful to include the vrf name in all these changes.

Reply:
I agree will add that change also.

I was trying to look at other protocols to see if there is a standard way to add the vrf name to logs.   For example should it be the first parameter in the log 

zlog_warn("VRF %s:  I/F %s HelloInterval mismatch: (my %d, rcvd %d)",